### ISSUE TITLE
recall and precision curve

### ISSUE THREAD

**Issue by mmoghadam11** [dingjiansw101/AerialDetection#79]
how can i get more evaluating without DOTA server(its just give AP):i need to get more like recall precision curve or APs APm ,...
after training it gives log.json file & with seaborn can get something like rbbox_loss_bbox , rbbox_loss_cls and acc , ...

1. what the meaning of acc ???how its defined in log
2. is there any way to find something like recall precision curve???

**Comment by dingjiansw101** [comment]
If you want to evaluate without DOTA server, you can only evaluate on the validation set instead of test set. You can also use the annotations of validation set to draw precision-recall curve by yourself.
